src/Reveal.Sdk.Dom/Data/DataSourceItems/WebServiceDataSourceItem.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Reveal.Sdk.Dom.Data
 {
-    internal class WebServiceDataSourceItem : DataSourceItem
+    public class WebServiceDataSourceItem : DataSourceItem
```

Comment:
a lot more work will be needed here. This will be very similar to the Rest data source item.